### PR TITLE
Adding scaling to e2e test scripts

### PR DESCRIPTION
**Pull request by dry923**
Adding scaling to the e2e test scripts

### CODE REVIEW

**Comment by rsevilla87** [comment]
Hey @dry923, it would be nice to use python virtual envs to install requirements. Same as Mohit is doing at https://github.com/cloud-bulldozer/e2e-benchmarking/pull/40

**Comment by dry923** [comment]
requires https://github.com/cloud-bulldozer/benchmark-operator/pull/409

**Comment by dry923** [comment]
> Hey @dry923, it would be nice to use python virtual envs to install requirements. Same as Mohit is doing at #40

actually the req's aren't even needed. it came from the folder i copied over to use as a skel. i'll remove it though. thanks!

File: workloads/scale-perf/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+export ES_SERVER_BASELINE=
+export ES_PORT_BASELINE=
+export BASELINE_UUID=
+export CERBERUS_URL=http://1.2.3.4:8080
+export SCALE=
+export POLL_INTERVAL=
+export POST_SLEEP=
+export TIMEOUT=
+export RUNS=
+#export GSHEET_KEY_LOCATION=
```
Comment (chaitanyaenr) [inline workloads/scale-perf/README.md:117]: Is GSHEET integration supported for scaling workload?

Comment (dry923) [reply workloads/scale-perf/README.md:117]: Sorry no. not at the moment. There's no touchstone role for it to compare so theres really nothing to push, lol. I'll pull that line and the other commented out run_compare one.

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by jtaleric — APPROVED** [review]
LGTM

**Pull request merged**